### PR TITLE
NXP-20931 Check id is not blank in getUserInfo

### PR DESCRIPTION
**Pull request by egiuly-nuxeo**
https://jira.nuxeo.com/browse/NXP-20931

[TestAndPush](https://qa.nuxeo.org/jenkins/view/TestAndPush/job/TestAndPush/job/ondemand-testandpush-egiuly/26/) succeeds

### CODE REVIEW

File: nuxeo-jsf/nuxeo-platform-webapp-base/src/test/java/org/nuxeo/ecm/webapp/security/UserSuggestionActionsBeanTest.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     egiuly
+ */
+package org.nuxeo.ecm.webapp.security;
+
+import static org.junit.Assert.*;
```
Comment (efge) [inline nuxeo-jsf/nuxeo-platform-webapp-base/src/test/java/org/nuxeo/ecm/webapp/security/UserSuggestionActionsBeanTest.java]: Don't use wildcards even for for static or test imports, for consistency.

**Comment by efge** [comment]
👍

File: nuxeo-jsf/nuxeo-platform-webapp-base/src/test/java/org/nuxeo/ecm/webapp/security/UserSuggestionActionsBeanTest.java (hunk 1 above)
Comment (egiuly-nuxeo) [inline nuxeo-jsf/nuxeo-platform-webapp-base/src/test/java/org/nuxeo/ecm/webapp/security/UserSuggestionActionsBeanTest.java]: @efge Good point, thanks.
So should I do:
```
import static org.junit.Assert
```
or 
```
import static org.junit.Assert.assertEquals;
import static org.junit.Assert.assertFalse;
import static org.junit.Assert.assertTrue;
```
?

Comment (atchertchian) [inline nuxeo-jsf/nuxeo-platform-webapp-base/src/test/java/org/nuxeo/ecm/webapp/security/UserSuggestionActionsBeanTest.java]: the latter

**Comment by atchertchian** [comment]
:+1:

**Pull request merged**